test/functional/mempool_package_onemore.py

```diff
@@ -48,10 +48,17 @@ def run_test(self):
         vout = utxo[0]['vout']
         value = utxo[0]['amount']
 
+        assert_equal(len(self.nodes[0].getrawmempool(True)), 0)
+
         fee = Decimal("0.0002")
         # MAX_ANCESTORS transactions off a confirmed tx should be fine
         chain = []
-        for _ in range(4):
+        # First tx has 3 outputs
+        (txid, sent_value) = self.chain_transaction(self.nodes[0], [txid], [vout], value, fee, 3)
+        vout = 0
+        value = sent_value
```

Comment:
redundant assignment in this loop (and the next one), can remove?

if not, may make sense to extract the loop body to a function